src/db/db.service.ts

```diff
@@ -11,24 +11,38 @@ export class DbConnectionService {
   private readonly logger = new Logger(DbConnectionService.name);
   constructor(private readonly configService: ConfigService) {}
 
-  async connect() {
-    this.logger.log('Establishing database connection ...');
+  private getConnection() {
+    return createConnection(this.configService.get<'string'>('DATABASE_URL'));
+  }
+
+  private loadMigrationConfig(dbInstance) {
```

Comment:
is the name `loadMigrationConfig` actually correct? What exactly is this function meant to do?